.gitignore

```diff
@@ -165,3 +165,4 @@ gulp-cache
 .rollupcache
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31892&commitId=f721e034dd346578210fb127b1d8250a0e26da98&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|6| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>